SU2_CFD/src/integration/CIntegration.cpp

```diff
@@ -137,15 +137,16 @@ void CIntegration::Space_Integration(CGeometry *geometry,
       case FAR_FIELD:
         solver_container[MainSolver]->BC_Far_Field(geometry, solver_container, conv_bound_numerics, visc_bound_numerics, config, iMarker);
         break;
-      case SYMMETRY_PLANE:
-        solver_container[MainSolver]->BC_Sym_Plane(geometry, solver_container, conv_bound_numerics, visc_bound_numerics, config, iMarker);
-        break;
       case ELECTRODE_BOUNDARY:
         solver_container[MainSolver]->BC_Electrode(geometry, solver_container, conv_bound_numerics, config, iMarker);
         break;
       case DIELEC_BOUNDARY:
         solver_container[MainSolver]->BC_Dielec(geometry, solver_container, conv_bound_numerics, config, iMarker);
         break;
+      case SYMMETRY_PLANE:
+        if (!config->GetNEMOProblem())
+          solver_container[MainSolver]->BC_Sym_Plane(geometry, solver_container, conv_bound_numerics, visc_bound_numerics, config, iMarker);
+        break;
```

Comment:
Why does the order of operations matter either for NEMO or the other solvers?
We don't have to play mikado with the regression tests, if the reference residuals change for some case but it converges to the same solution we update the regression.

Reply:
See also my answer to one of your comments below in `CNEMOEulerSolver.cpp` .